### DISCUSSION TITLE
Release discussions?

### DISCUSSION THREAD

**Discussion by bl-ue** [nlohmann/json#2741] [General]
Hey @nlohmann! 👋🏻 It would be really cool if you created a discussion when you created releases, 

<img width="469" alt="Screen Shot 2021-04-25 at 8 48 08 AM" src="https://user-images.githubusercontent.com/54780737/115993937-f940f180-a5a2-11eb-8557-5d0fdbd23a1d.png">

You can see an example of it [here](https://github.com/microsoft/winget-cli/releases/tag/v-0.3.11102-preview).

**Comment by nlohmann** [comment]
I was not aware of this feature, but I will definitely give it a try with the next release!

**Reply by bl-ue** [reply]
Btw, do you have an ETA for the next release?  I'm a big fan of this library, and I use it every day, and I'd love to see  new features and bug fixes incorporated.

**Reply by bl-ue** [reply]
ping @nlohmann did thing this get lost in your notifications?

**Reply by nlohmann** [reply]
I don't have an ETA right now. The pandemic makes it hard to plan these days...

**Reply by bl-ue** [reply]
No kidding! 😄  No problem. 👍🏻  Thank you!

**Reply by nlohmann** [reply]
Done: https://github.com/nlohmann/json/releases/tag/v3.10.0 -> now with a release discussion, see https://github.com/nlohmann/json/discussions/2937

**Comment by coolLimpan** [comment]
Hi 
I'm looking into using this library. What would be your recommendation? go with the latest release or latest dev version (important bugfixes or other)?

**Reply by nlohmann** [reply]
I'd take the released version unless you run into any trouble with it. The develop version is basically bleeding edge and may break existing things temporarily.
